tests/foreman/rhai/conftest.py

```diff
@@ -18,7 +18,9 @@ def module_org():
 
     :rtype: :class:`nailgun.entities.Organization`
     """
-    return nailgun.entities.Organization(id=DEFAULT_ORG_ID).read()
+    default_org_id = nailgun.entities.Organization().search(
+        query={'search': 'name="{}"'.format(DEFAULT_ORG)})[0].i
```

Comment:
you missed 'd' here 

```suggestion
        query={'search': 'name="{}"'.format(DEFAULT_ORG)})[0].id
```

Reply:
Ahh ... It was a TYPO :) ... pushed in new commit .